Mathlib/Data/Complex/Exponential.lean

```diff
@@ -1974,7 +1974,18 @@ theorem exp_bound_div_one_sub_of_interval' {x : ℝ} (h1 : 0 < x) (h2 : x < 1) :
       _ = 1 - (1 + x + x ^ 2) * (1 - x) := by ring
   calc
     exp x ≤ _ := exp_bound' h1.le h2.le zero_lt_three
-    _ ≤ 1 + x + x ^ 2 := by norm_num [Finset.sum] <;> nlinarith
+    _ ≤ 1 + x + x ^ 2 := by
+      -- Porting note: was `norm_num [Finset.sum] <;> nlinarith`
+      rw [Finset.sum, range_val]
+      nth_rw 1 [← two_add_one_eq_three]
+      rw [← Nat.succ_eq_add_one, Multiset.range_succ, Multiset.map_cons, Multiset.sum_cons]
+      nth_rw 3 [← one_add_one_eq_two]
+      rw [← Nat.succ_eq_add_one, Multiset.range_succ, Multiset.map_cons, Multiset.sum_cons]
+      nth_rw 3 [← zero_add 1]
+      rw [← Nat.succ_eq_add_one, Multiset.range_succ, Multiset.map_cons, Multiset.sum_cons]
+      rw [Multiset.range_zero, Multiset.map_zero, Multiset.sum_zero]
+      norm_num
+      nlinarith
```

Comment:
Actually, we should probably work out what happened here before merging

Reply:
Oh, I bet we're missing [the `norm_num` plugin for big operators](https://github.com/leanprover-community/mathlib/blob/master/src/algebra/big_operators/norm_num.lean). @Vierkantor, do you know if this has been ported?

Reply:
@eric-wieser [No.](https://leanprover-community.github.io/mathlib-port-status/file/algebra/big_operators/norm_num)

Reply:
Also possible that the positivity extensions in the not-yet-merged #3907 are needed.

Reply:
I'd be happy to port the plugin, and actually have some ideas on how to improve the implementation, anyone willing to give me a quick course on Lean4 tactics? :)

Reply:
@Vierkantor Yes, let's do that together later today

Reply:
But I think we should definitely not let this PR wait for the port of the norm-num extension. We can clean up the proofs after that is done, because some porting PRs are waiting for this PR.

Reply:
I agree; though we should probably explain in the porting note that the old proof can be restored once we port the missing tactic.

Reply:
What's the status here?

Reply:
I've added the porting note. We can fix this later today when Anne is ready. :-)